internal/service/elasticache/replication_group.go

```diff
@@ -461,6 +474,14 @@ func resourceReplicationGroupCreate(d *schema.ResourceData, meta interface{}) er
 		params.CacheParameterGroupName = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("ip_discovery"); ok {
+		req.IpDiscovery = aws.String(v.(string))
+	}
+
+	if v, ok := d.GetOk("network_type"); ok {
+		req.NetworkType = aws.String(v.(string))
```

Comment:
use `params` instead of `req`

```suggestion
		params.NetworkType = aws.String(v.(string))
```

Reply:
Completed